assets/src/scripts/components/dailyValueHydrograph/legend.js

```diff
@@ -0,0 +1,94 @@
+// functions to facilitate DV legend creation for a d3 plot
+import {createStructuredSelector} from 'reselect';
+
+import {getLayout} from './selectors/layout';
+import {getLegendMarkerRows} from './selectors/legend-data';
+
+import config from '../../config';
+import { mediaQuery } from '../../utils';
```

Comment:
Consider making this consistant with the other imports, with no space after and before the brackets.